### ISSUE TITLE
Air Quality App

### ISSUE THREAD

**Issue by matthardwick** [Slimfit/climatepatch#8]
I was watching _Before the Flood_ and DiCaprio talked to a person in China about an app that was developed that essentially showed a map of the user's location and the nearby polluters, along with a color-coded scale of some sort to show the severity of their pollution. While it might not be as effective in the States, I figured something similar would be a decent idea, to allow people to see exactly who is responsible for the pollution around them. 

Does something like this already exist outside of China?

**Comment by indigocarmen** [comment]
https://itunes.apple.com/us/app/global-air-quality-monitoring/id1048912974?mt=8

That may be something...There is actually a lot of apps that do something similar. Should we team up with one instead of developing a new one? Perhaps use their data to populate a map on the website?

**Comment by jgoslow** [comment]
If we can get to an API then it would be one of many things that could be mapped.  One thing to remember too is many organizations would be happy to share their information, especially if we're giving credit to them for the data.  Not sure if there's any open data for this sort of thing - but it could also be self-reported...  sort of like the weather... capitalize on the existing network of people paying attention to weather already.

**Comment by indigocarmen** [comment]
I do kind of like the self reporting functionality, similar to Wayz. However, how would users know...? I have no clue what the air quality is like around me.

**Comment by jgoslow** [comment]
Learn to report air quality and EARN YOUR PATCH!

**Comment by VictorPeralta** [comment]
An Arduino could be an option for air quality testing, since this is a tech-oriented community it wouldn't be surprising if a few people already had knowledge of them. And if they don't, it's cheap and easy to understand. Something like this is what I mean: http://www.instructables.com/id/Air-Quality-Test-Box/

**Comment by indigocarmen** [comment]
Oh that's actually pretty nifty! We could give out one of the 'gold' level patches for building something like that? maybe down the road actually sell those units?

**Comment by VictorPeralta** [comment]
Yup, I think it wouldn't be hard to create this hardware solution and we could probably sell it later on.

**Comment by stoelsy** [comment]
> showed a map of the user's location and the nearby polluters

Would be interesting to see an overview of all the factories/heavy polluters in your area and the associated air quality.
